packages/sku/src/context/createSkuContext.ts

```diff
@@ -20,6 +20,26 @@ const jiti = createJiti(__filename);
 
 const debug = _debug('sku:config');
 
+const generateTypeScriptPaths = (
+  pathAliases?: Record<string, string>,
+): Record<string, string[]> => {
+  const typeScriptPaths: Record<string, string[]> = {};
+
+  // Always include default src/* alias, then merge with user-provided aliases
+  const mergedAliases = {
+    ...defaultSkuConfig.pathAliases,
```

Comment:
I think we might actually not want to include the `src/*` default in `defaultSkuConfig.pathAliases`. `defaultSkuConfig` is really meant to hold default values for `undefined` fields, not be merged into user-provided values.

So rather than specifying `{ 'src/*': './src/*' }` as the default (both in `defaultSkuConfig` and in the docs), we should instead just say "you always get this mapping no matter what", and merge it in manually.